thread_pthread.c

```diff
@@ -3081,7 +3094,7 @@ rb_thread_create_timer_thread(void)
 
             CLOSE_INVALIDATE_PAIR(timer_th.comm_fds);
 #if HAVE_SYS_EPOLL_H && USE_MN_THREADS
```

Comment:
I originally made this `#if (HAVE_SYS_EPOLL_H || HAVE_SYS_EVENT_H) && USE_MN_THREADS`, but several specs failed with `EBADF` errors. In the docs (https://man.openbsd.org/kqueue.2) it says `The queue is not inherited by a child created with [fork(2)](https://man.openbsd.org/fork.2). Similarly, kqueues cannot be passed across UNIX-domain sockets.`. Possibly that is the reason it's invalid at this point?